### ISSUE TITLE
Accessibility issues in the new 2FA UI [bz1463692]

### ISSUE THREAD

**Issue by bug-mirror** [mozilla/fxa-bugzilla-mirror#504]
From https://bugzilla.mozilla.org/show_bug.cgi?id=1463692

STR:
1. Go to https://accounts.firefox.com/settings
2. Enable two step verification.

First issue: The QR code has no alternative text. That way, assistive technologies cannot find it and thus a blind user using either a touch screen PC or otherwise getting oriented via their screen reader cannot find where to point their smartphone to scan the QR code with the authenticator app.
This is not a decorative image, so should have an alternative text that is meaningful.

3. After scanning the bar code, and entering the resulting verification number, the backup codes appear.

Second problem: The links to download, copy etc. are not exposed as links for assistive technologies. For one, conceptually, these are buttons rather than links, since they don't actually open a new page, but perform an action, much like the Finish button at the bottom. Second, I do not know what it is in the markup, probably a missing href attribute, that is making these not links, but plain text for assistive technologies.
